ios/Classes/FlutterCarplayPlugin.swift

```diff
@@ -501,6 +505,17 @@ public class FlutterCarplayPlugin: NSObject, FlutterPlugin {
 
             result(false)
 
+        case FCPChannelTypes.openExternalMap:
+            guard let args = call.arguments as? [String: Any],
+                  let url = args["url"] as? String
+            else {
+                result(false)
+                return
+            }
+            if let formattedUrl = URL(string: url) {
+               FlutterCarplayTemplateManager.shared.carplayScene?.open(formattedUrl, options: nil, completionHandler: nil)
```

Comment:
is `result(true)` missing?

Reply:
added the result in the completion handler